Makefile

```diff
@@ -21,20 +25,54 @@ configure:
 	echo "JIT_CLIENT_SECRET=$$client_secret" >> .env; \
 	echo "GITHUB_API_TOKEN=$$github_token" >> .env; \
 	echo "TEAM_WILDCARD_TO_EXCLUDE=$$topics_to_exclude" >> .env
-
-create-teams:
+endif
+ifeq ($(filter run,$(MAKECMDGOALS)),run)
 	. venv-jit/bin/activate && \
 	export PYTHONPATH=$(CURDIR) && \
-	 python3 src/utils/github_topics_to_json_file.py && \
-	  python3 src/scripts/create_teams.py teams.json
+	python3 src/utils/github_topics_to_json_file.py && \
+	python3 src/scripts/create_teams.py teams.json
+endif
+
+install:
+	@echo installation complete
+configure:
+	@echo configuration complete
+run:
+	@echo run complete
+
+
+SELF_HOSTED_DOCKER_CENTOS_SCRIPT := src/scripts/self-hosted-runners/setup-self-hosted-runner-centos.sh
+SELF_HOSTED_DOCKER_UBUNTU_SCRIPT := src/scripts/self-hosted-runners/setup-self-hosted-runner-ubuntu.sh
+SELF_HOSTED_RUNNER_SCRIPT := src/scripts/self-hosted-runners/install-github-runner-agent.sh
+
+
+self-hosted-runner: check-root setup-runner install-agent
+
+check-root:
+	@if [ "$$UID" -eq 0 ]; then \
+        echo "Error: This script should not be run as root."; \
+        exit 1; \
+    fi
+
+setup-runner:
+ifeq ($(filter centos,$(MAKECMDGOALS)),centos)
+	sudo yum install -y jq
+	chmod +x $(SELF_HOSTED_DOCKER_CENTOS_SCRIPT)
+	./$(SELF_HOSTED_DOCKER_CENTOS_SCRIPT)
+else ifeq ($(filter ubuntu,$(MAKECMDGOALS)),ubuntu)
```

Comment:
are we sure we dont need jq installation at ubuntu?

Reply:
works with clean ubuntu ami on ec2